ground/src/main/java/com/google/android/ground/persistence/local/LocalValueStore.kt

```diff
@@ -45,6 +45,13 @@ class LocalValueStore @Inject constructor(private val preferences: SharedPrefere
       preferences.edit().putInt(MAP_TYPE, type).apply()
     }
 
+  /** State of the gps lock. */
```

Comment:
Please replace "GPS" with "location" here and throughout, since Android fused location service derives location from other sources in addition to GPS.

Reply:
Done